docs/api/deprecations.md

```diff
@@ -0,0 +1,54 @@
+---
+id: deprecations
+title: Deprecations
+description: A list of active and past deprecations
+---
+
+## Introduction
+
+This page contains extended documentation for some of the deprecations in
+various parts of Backstage. It is not an exhaustive list as most deprecation
+only come in the form of a changelog notice and a console warning. The
+deprecations listed here are the ones that need a bit more guidance than what
+fits in a console message.
+
+### App Theme
```

Comment:
One thing that's potentially missing is some version that's affected by this deprecation.
It makes sense getting to this page by visiting the link from the deprecation warning but how should a user interpret this page when causally browsing backstage.io?

Reply:
Yeah true, and date as well, will add them in